en/SUMMARY.md

```diff
@@ -61,6 +61,9 @@
 * Middleware and Architecture
   * [uORB Messaging](middleware/uorb.md)
   * [MAVLink Messaging](middleware/mavlink.md)
+  * [DDS/ROS2 interface](middleware/micrortps.md)
```

Comment:
Is there a better name than this? Perhaps "PX4/RTPS Bridge" or "RTPS DDS"  Not sure, but I wouldn't see this and think "hey yes, that's how I publish uORB topics externally"